### DISCUSSION TITLE
ZMNHVD Qubino / Flush Dimmer 0-10V is missing General Purpose value

### DISCUSSION THREAD

**Discussion by vilellic** [zwave-js/zwave-js#6582] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
ZMNHVD Qubino / Flush Dimmer 0-10V is configured as General Purpose sensor (parameter #1: 5). In previous versions I got the General Purpose reading from the input pins. But it disappeared in some version in the past (maybe within couple of months). I noticed it after updating the new Z-Wave JS UI version.

I use this dimmer to control my house ventilation unit. I can control the ventilation speed with output 0-10V and previously got the reading what was the speed in use (also 0-10V).

I have tried to re-interview it many times with no luck.

I have only this left as evidence that I got the value in the past (in Home Assistant) but now it says it's no longer available and I can't see the parameter in Z-Wave JS UI neither.
<img width="315" alt="Näyttökuva 2024-1-1 kello 23 30 41" src="https://github.com/zwave-js/node-zwave-js/assets/15627846/e067dfc2-0d72-450d-a289-14d7979c134f">

### Software versions

zwave-js-ui: 9.6.2.6e369a1
zwave-js: 12.4.1

zwave adapter: Razberry in raspberry 4

### Device information

Manufacturer: Qubino
Model name: ZMNHVD Flush Dimmer 0-10V
Node ID: 020

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-01-15.log](https://github.com/zwave-js/node-zwave-js/files/13976680/zwavejs_2024-01-15.log)

[a comment by an automated account is not shown]

**Reply by vilellic** [reply]
Uploaded new log file

**Comment by AlCalzone** [comment]
What the bot said. Can't help without the correct log.

**Comment by vilellic** [comment]
Hey @AlCalzone , and sorry for the wrong log. Here is the correct one with interview (node 020) and fiddling with the dimmer value up / down.
[zwavejs_2024-01-15.log](https://github.com/zwave-js/node-zwave-js/files/13941782/zwavejs_2024-01-15.log)

**Reply by AlCalzone** [reply]
In your log the device claims that it does not support multiple endpoints, but then later sends commands from those endpoints.

Since this is a Qubino device, I'm assuming you changed the input type, but forgot to exclude and re-include the device afterwards like the manual tells you to.

**Comment by vilellic** [comment]
@AlCalzone Could you check this one again?
